Source/GR32_Backends_LCL_Gtk.pas

```diff
@@ -174,6 +176,8 @@ procedure TLCLBackend.NeedCanvas;
 
 procedure TLCLBackend.CopyCanvasToPixmap;
 begin
+  NeedCanvas; // avoid error: FBitmap is nil.

```

Comment:
This should not be necessary. Are you saying that you experienced that `FBitmap` was **nil** here?

`CopyCanvasToPixmap` is only called from `NeedBits` if `FCanvasDirty` is **True**.
`FCanvasDirty` only get set **True** in `CanvasChanged` which is called from the `TCanvas.OnChange` handler.
`TCanvas.OnChange` can not be called unless the `TBitmap` has been created (and assigned to `FBitmap`).